spec/integration/guest_view_spec.rb

```diff
@@ -84,6 +85,14 @@
     end
   end
 
+  describe "prohibited paths" do
+    it "redirects the guest to the root when attempting to visit orders_path" do
+      visit orders_path
+
+      expect(current_path).to eq(root_path)
+    end
+  end
+
   def create_item
     image = create(:image)
     item = FactoryGirl.create(:item, title: "milk",
```

Comment:
You shouldnt need FactoryGirl  here, you can just call create, did you merge in latest master?